modbus_cli/definitions.py

```diff
@@ -1,7 +1,7 @@
 import re
 import logging
 
-REGISTER_RE = re.compile(r'([cCdhHi]@)?(\d+)(/[^:|]*)?([:|].*)?')
+REGISTER_RE = re.compile(r'^([cCdhHi]@)?(\d+|0x\d+)(/[^:|]*)?([:|].*)?$')
```

Comment:
I think `\d' only matches `[0-9]` not `[a-f]`

Reply:
damn! too quick :)